src/dnd/mod.rs

```diff
@@ -0,0 +1,14 @@
+use crate::Clipboard;
+
+impl Clipboard {
+    /// Set up DnD operations for the Clipboard
+    pub fn init_dnd() {}
+
+    /// Start a DnD operation on the given surface with some data
+    pub fn start_dnd() {}
+
+    /// End the current DnD operation, if there is one
+    pub fn end_dnd() {}
+
+    
+}
```

Comment:
New line in the end of file.

Reply:
Ahh, this file will be part of a separate PR for DnD. So I'll remove it.